code/genesis_call.dme

```diff
@@ -50,3 +50,12 @@
  */
 /world/proc/_()
 	var/static/_ = world.Genesis()
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+// END_INCLUDE
```

Comment:
?

Reply:
I honestly have no idea, I didn't touch that file

Reply:
@Shadow-Quill do you want me to just delete those additions?